internal-services/catalog/create-advisory-task.yaml

```diff
@@ -101,6 +102,8 @@ spec:
           ADVISORY_NUM=$(((RND=RANDOM<<15|RANDOM)) ; echo ${RND: -6})
 
           YEAR=$(date '+%Y')
+          # Use ISO 8601 format in UTC/Zulu time, e.g. 2024-03-06T17:27:38Z
+          SHIP_DATE=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Should $YEAR then be extracted from this? To prevent the super super rare edge case? :sunglasses:

Reply:
I had a similar concern with the year, but there the gap between defining the year and actually pushing the commit could definitely be bigger, because you're calling an api. Here it's really just milliseconds, so I'd say it's ok. 

But just for you... done 😎

Reply:
:joy: